ergotree-interpreter/src/eval/sglobal.rs

```diff
@@ -98,4 +196,98 @@ mod tests {
             expected_xor.as_slice()
         );
     }
+
+    #[test]
+    fn eval_from_bigendian_bytes() {
+        let type_args = std::iter::once((STypeVar::t(), SType::SInt)).collect();
+
+        let bytes = vec![0_i8, 0, 0, 1];
+        let expr: Expr = MethodCall::new(
+            Expr::Global,
+            sglobal::FROM_BIGENDIAN_BYTES_METHOD
+                .clone()
+                .with_concrete_types(&type_args),
+            vec![bytes.into()],
+        )
+        .unwrap()
+        .into();
+        let ctx = force_any_val::<Context>();
+        assert_eq!(eval_out::<i32>(&expr, &ctx), 1);
+    }
+
+    #[test]
+    fn eval_from_bigendian_bytes_short() {
+        let type_args = std::iter::once((STypeVar::t(), SType::SShort)).collect();
+
+        let bytes = vec![0_i8, 1];
+        let expr: Expr = MethodCall::new(
+            Expr::Global,
+            sglobal::FROM_BIGENDIAN_BYTES_METHOD
+                .clone()
+                .with_concrete_types(&type_args),
+            vec![bytes.into()],
+        )
+        .unwrap()
+        .into();
+        let ctx = force_any_val::<Context>();
+        assert_eq!(eval_out::<i16>(&expr, &ctx), 1);
+    }
+
+    #[test]
+    fn eval_from_bigendian_bytes_long() {
+        let type_args = std::iter::once((STypeVar::t(), SType::SLong)).collect();
+
+        let bytes = vec![0_i8, 0, 0, 0, 0, 0, 0, 1];
+        let expr: Expr = MethodCall::new(
+            Expr::Global,
+            sglobal::FROM_BIGENDIAN_BYTES_METHOD
+                .clone()
+                .with_concrete_types(&type_args),
+            vec![bytes.into()],
+        )
+        .unwrap()
+        .into();
+        let ctx = force_any_val::<Context>();
```

Comment:
Consider using eval_out_wo_ctx instead for clarity

Reply:
Done.